src/controllers/controller.line.js

```diff
@@ -243,13 +243,16 @@ module.exports = DatasetController.extend({
 
 	updateBezierControlPoints: function() {
 		var me = this;
+		var chart = me.chart;
 		var meta = me.getMeta();
-		var area = me.chart.chartArea;
-		var points = (meta.data || []);
+		var lineModel = meta.dataset._model;
+		var area = chart.chartArea;
```

Comment:
I might call this `chartArea` to be a little clearer. And I think `model` would probably be fine for the previous variable. But I don't feel strongly about either

Reply:
`model` is used to keep `point._model` later, so this needs to be a different name.